.gitignore

```diff
@@ -49,7 +49,6 @@ lib/
 
 # cmake
 CMakeCache.txt
-*.cmake
```

Comment:
This needs to stay there. Force add your other cmake files instead.
Running cmake once generates 26 new .cmake files

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)

        JSStandAlone/cmake_install.cmake
        cmake_install.cmake
        samples/Animations/cmake_install.cmake
        samples/Components/cmake_install.cmake
        samples/Crypto/cmake_install.cmake
        samples/Cursor/cmake_install.cmake
        samples/CustomMesh/cmake_install.cmake
        samples/Dungeon/cmake_install.cmake
        samples/Effects/cmake_install.cmake
        samples/Empty/cmake_install.cmake
        samples/Entities/cmake_install.cmake
        samples/GamePads/cmake_install.cmake
        samples/Http/cmake_install.cmake
        samples/Multiplayer/cmake_install.cmake
        samples/Navigation/cmake_install.cmake
        samples/Particles/cmake_install.cmake
        samples/Primitives/cmake_install.cmake
        samples/Random/cmake_install.cmake
        samples/Shader/cmake_install.cmake
        samples/Sounds/cmake_install.cmake
        samples/SpriteFrames/cmake_install.cmake
        samples/Sprites/cmake_install.cmake
        samples/Text/cmake_install.cmake
        samples/Textures/cmake_install.cmake
        samples/TiledMap/cmake_install.cmake
        samples/UI/cmake_install.cmake
```